src/utils/data.ts

```diff
@@ -191,18 +191,45 @@ export const createZipFromObject = (obj: unknown): Promise<Uint8Array> => {
   );
 };
 
+export const reverseSafeSchema = function (
+  schema: Array<Record<'id', string>>
+) {
+  return schema.reduce((accu, curr) => {
```

Comment:
maybe change accu, and curr to a more understandable Naming: 
accumulator, currentElement for example